tests/lib/rules/no-ember-data-global-imports.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-ember-data-global-imports');
+const RuleTester = require('eslint').RuleTester;
+const parserOptions = { ecmaVersion: 6, sourceType: 'module' };
+
+const message = 'Imports from @ember-data packages should be prefered over imports from ember-data';
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+ruleTester.run('no-ember-data-global-imports', rule, {
+  valid: [
+    {
+      code: `import Model from '@ember-data/model';`,
+      parserOptions,
+    },
+    {
+      code: `import Model, { attr } from '@ember-data';
+
+        export default Model.extend({
+          name: attr('string')
+        });
+      `,
+      parserOptions,
+    },
```

Comment:
You can just have an array of strings for the `valid` test cases, no need to pass an object for each.

```
valid: [
  'import ...',
  'import ...'
]